charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v0.15.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 6.0.1
+version: 6.0.2
```

We have to consider this change a minor one as a new feature has ben introduced.
```suggestion
version: 6.1.0
```

---

Ah yeah, you said minor, not patch. Sorry, my bad. Committing the change.